modules/Bio/EnsEMBL/Analysis/Hive/Config/long_read.pm

```diff
@@ -0,0 +1,589 @@
+#!/usr/bin/env perl
```

Comment:
This is not needed because it is not a script and should be removed

Reply:
Deleted; it was added to prevent perltidy with the previous settings generate an empty line at the start of the document.